controllers/users.js

```diff
@@ -33,34 +57,28 @@ const registerUser = async (req, res, next) => {
   }
 };
 
-const loginUser = async (req, res, next) => {
+router.post("/register", registerUser);
+
+router.patch("/avatars", upload.single("avatar"), async (req, res, next) => {
```

Comment:
functions INSIDE router functions shouldnt be initialized here. Initialize it in controller